src/leap/mail/imap/mailbox.py

```diff
@@ -514,13 +514,14 @@ def _get_imap_msg(messages):
                 d_imapmsg = []
                 for msg in messages:
                     d_imapmsg.append(getimapmsg(msg))
-                return defer.gatherResults(d_imapmsg)
+                return defer.gatherResults(d_imapmsg, consumeErrors=True)
```

Comment:
One day I will understand how `consumeErrors` works. Almost every time I set it up by a try-and-error process, and usually it is the opposite value of what I was expecting.